extensions/ql-vscode/src/queryserver-client.ts

```diff
@@ -171,6 +171,22 @@ export class QueryServerClient extends DisposableObject {
       args.push('--old-eval-stats');
     }
 
+    if (await this.cliServer.cliConstraints.supportsStructuredEvalLog()) {
+      if (this.config.structuredEvalLogFile) {
+        args.push('--evaluator-log');
+        args.push(this.config.structuredEvalLogFile);
```

Comment:
We should make sure the structured logs go in the same directory as all of the other query artifacts. I don't see much of a reason to allow users to set the name. But, we _do_ want to create a new command that allows users to open and view the logs (like we do for the regular log). See `codeQLQueryHistory.showQueryLog` command.

Reply:
It's a little more complicated since I see that there is a `codeQL.runningQueries.customLogDirectory` setting. I'm not even sure if that's working any more.

Reply:
I have updated the PR to create the file in the same directory as the other query artifacts. When the change to the CLI is made to create logfiles per query (see PR description), we can create a directory for the structured log files (potentially a custom one as well). 

After we are able to consume the individual logfiles per query, we could add the open/view log command.